public/app/features/explore/state/selectors.test.ts

```diff
@@ -49,60 +49,60 @@ describe('Deduplication selector', () => {
     // Problem was caused by mutating the log results entries in redux state. The memoisation hash for deduplicatedLogsSelector
     // was changing depending on duplicates information from log row state, while should be dependand on log row only.
 
-    let dedups = deduplicatedLogsSelector(state as ExploreItemState);
-    expect(dedups.rows.length).toBe(10);
+    let dedups = deduplicatedRowsSelector(state as ExploreItemState);
```

Comment:
Not sure if this test actually makes sense right now. Not sure what the memozation strategy was before but right now there should not be a logical case where this should fail.

Reply:
Yeah, should be rewritten to cover main use case.